package.json

```diff
@@ -130,6 +130,22 @@
     "web-vitals": "^1.0.1",
     "yup": "^0.32.11"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js, jsx}": [
```

Comment:
this won't work. These aren't js or jsx files they're ts and tsx. Are you sure you tested this? By test I mean that after committing the files became formatted and had previously been unformatted

Reply:
Fac, ye but, as i said the struggle was with making sure i disabled prettier first. Maybe i somehow thoiught i did, but didnt, if you understad